src/main/resources/openapi/token-1.0.yaml

```diff
@@ -107,6 +106,40 @@ paths:
           $ref: "#/components/responses/trait_400"
         "500":
           $ref: "#/components/responses/trait_500"
+  /token/logout:
```

Comment:
The logout routes here in mod-authtoken are called by the logout routes in mod-logout. Each has its own purpose. In the case of mod-login, the purpose of the logout routes are to respond with `Set-Cookie` with a time in the past to remove the cookie on the client. The purpose of the logout routes in mod-authtoken are to perform the actual removal of the refresh token from storage to ensure that it cannot be used to request new access tokens.

In other words: mod-login has responsibility for cookies and mod-authtoken has responsibility for storage.

Reply:
> The logout routes here in mod-authtoken are called by the logout routes in mod-logout. Each has its own purpose. 

I agree that they have different purposes.

To me, they operate at different levels.

> In the case of mod-login, the purpose of the logout routes are to respond with `Set-Cookie` with a time in the past to remove the cookie on the client. The purpose of the logout routes in mod-authtoken are to perform the actual removal of the refresh token from storage to ensure that it cannot be used to request new access tokens. In other words: mod-login has responsibility for cookies and mod-authtoken has responsibility for storage.

Ok.

Could this be named to refer to the tokens being removed or invalidated, rather than logging in / out?

Reply:
I'm not against this if you think it would be an improvement. What would you prefer? Could do `/token/invalidate` and 'token/invalidate-all`. That might make it a bit closer semantically to what is happening. I think I would prefer "invalidate" over "remove" since the data deletion seems somewhat lower down whereas "validation/invalidation" is the broader concept and it is a term that is used throughout the source code for this feature. "Revoke" might be another candidate but it doesn't have convenient a opposite like "validate/invalidate".

Reply:
> I think I would prefer "invalidate" over "remove" since the data deletion seems somewhat lower down whereas "validation/invalidation" is the broader concept and it is a term that is used throughout the source code for this feature. 

I'm good with this reasoning. You are close to the domain and understand the semantic intent of the operation.

I think the term being used throughout the source is a good indicator.

Reply:
Renamed /logout routes to /invalidate.